src/guides/libraries/c-interop.md

```diff
@@ -253,6 +253,58 @@ To reduce this burden,
 you can use the [`package:ffigen`][ffigen] binding generator
 to automatically create FFI wrappers from C header files.
 
+## Building and bundling native assets
+
+{{site.alert.note}}
+  The native assets feature is **experimental**,
+  and [in active development](https://github.com/dart-lang/sdk/issues/50565).
+{{site.alert.end}}
+
+The Native Assets feature aims to resolve a number of issues associated with the
+distribution of Dart packages that depend on native code. It does so by
```

Comment:
The Native Assets feature aims to make it pleasant for Dart packages to depend on and use native code:

* It packages to include native code (in source or binary form)
* It builds (if needed) the native code or obtains the binaries via a package's `build.dart` file
* It automatically bundles native code into the app
* It automatically makes the native code available at runtime via declarative `@Native<>() extern` syntax

All of this works in standalone dart as well as flutter.